src/components/ContentSidebar/ActivityFeed/activity-feed/__tests__/__snapshots__/ActivityFeed-test.js.snap

```diff
@@ -8,7 +8,7 @@ exports[`components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed shoul
   <div
     className="bcs-activity-feed-items-container"
   >
-    <ActiveState
+    <Component
```

Comment:
This seems strange. Any ideas?

Reply:
This happens when you use the higher order component for error handling. It wraps the component and inserts an inlineError or a maskError @DanDeMicco